web/src/beta/features/SidePanelCard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import type { PropsWithChildren } from "react";
+
+import { styled, useTheme, Theme } from "@reearth/services/theme";
+
+type Props = {
+  title: string;
+};
+
+const SidePanelCard: React.FC<PropsWithChildren<Props>> = ({ title, children }) => {
```

Comment:
Avoid PropsWithChildren
See what dan abramov (one of the main developers on React) says about it [here](https://stackoverflow.com/a/71809927/13699142)

Reply:
Thank you, [fixed!](https://github.com/reearth/reearth/pull/463/commits/05369e0ce424c0ed53c608138ec2d9b1ddfed827)